modules/benchmark/R/load.netcdf.R

```diff
@@ -1,29 +1,72 @@
-load.netcdf <- function(path,vars,units,newunits){
-  
-  print(paste("Years: ",start.year," - ",end.year),sep="")
-  result <- list()
-  for(ncfile in ncfiles) {
-    nc <- nc_open(ncfile)
-    for(i in i:length(vars)){
-      v <- vars[i]
-      u1 <- units[i]
-      u2 <- newunits[i]
-      if(v %in% c(names(nc$var),names(nc$dim))){
-        newresult <- ncvar_get(nc, v)
-        newresult <- ud.convert(newresult, u1, u2)
-        result[[v]] <- abind(result[[v]], newresult)
-      } else if (!(v %in% names(nc$var))){
-        logger.warn(paste(v, "missing in", ncfile))
-      }
+##' @name load.x_netcdf
+##' @title load.x_netcdf
+##' @export
+##' @param data.path character vector or list
+##' @param format list
+##' @param start_year numeric
+##' @param end_year numeric
+##' @param site list
+##' @param vars character
+
+load.x_netcdf <- function(data.path, format, site, vars=NULL){
```

Comment:
`site` argument is not used in this function, but this has something to do with @bcow and @mdietze's design for the future use of this framework